Tests/test_PDB_Mutate.py

```diff
@@ -0,0 +1,32 @@
+"""Tests for PDB Mutate module."""
+
+import unittest
+from Bio.PDB.Mutate import mutate
+from Bio.PDB import PDBParser
+from Bio.PDB import Mutate
+
+
+class MutateTest(unittest.TestCase):
+    parser = PDBParser(QUIET=True)
+    structure = parser.get_structure("my_structure", "Mutate/5e0m.pdb")
+    mutate(structure, chain="A", mutate_to="TYR", res_num=44, mutation_type="best")
```

Comment:
You probably want the entry point to this method to either be a class e.g. `RotamerSampler` that takes a structure and acts on a residue, kind of like the Superimpose works. That way you can load the rotamer library once (more efficient) and then do all sorts of manipulations. You might also want to pass a `Residue` object to the method, instead of a chain + resnum combo, as that's more robust and will play nicely with the API @Will-Tyler is building for selection.